_includes/head.html

```diff
@@ -16,4 +16,5 @@
   <link rel="alternate" type="application/rss+xml" title="{{ site.title | escape }}" href="{{ "/feed.xml" | relative_url }}">
   <link rel="shortcut icon" href="/assets/favicon.png" type="image/x-icon">
   <link rel="icon" href="/assets/favicon.png" type="image/x-icon">
+  <script async defer data-domain="techworkerscoalition.org" src="https://plausible.io/js/plausible.js"></script>
```

Comment:
i approved but you don't need both async and defer, just one or the other

Reply:
old IE browsers don't recognize async, in which case defer is used. Otherwise, Async always ranks higher. I can remove Defer if we don't want to support/text IE (which truthfully, without this....plugin, would have no idea what our users are doing)

Reply:
sure but i mean... are our users really hitting the twc site with an old version of IE?